src/Umbraco.Community.Contentment/DataEditors/Buttons/buttons.html

```diff
@@ -10,7 +10,7 @@
                 ng-repeat="item in vm.items"
                 ng-class="{ 'umb-button--selected' : item.selected }"
                 label="{{vm.hideName === false ? item.name : null}}"
-                title="{{item.name}}"
+                title="{{item.name}}, {{item.description}}"
```

Comment:
@enkelmedia Should the 'name' part display if the button label is visible?

The logic may need to be like this...?

```
title="{{vm.hideName === false ? item.description : item.name + ', ' + item.description}}"
```

Reply:
This would work but would add a `,` after the name if the description is empty =D

Reply:
@enkelmedia Ah yes, doh! 😅  I'll leave it with you. 🤘